cypress/e2e/michael.cy.ts

```diff
@@ -17,5 +17,28 @@ describe('obstacle practice', () => {
         cy.visit('https://obstaclecourse.tricentis.com/Obstacles/60469')
         cy.get('#toscabot').dragTo('#to')
         cy.contains('Good job!')
-    })
+    });
+
+    it('41038', () => {
+        cy.visit('https://obstaclecourse.tricentis.com/Obstacles/41038')
+        cy.contains('Click into my right half').click('right', { force: true })
+        cy.contains('Good job!')
+    });
+
+    it('41037', () => {
+        cy.visit('https://obstaclecourse.tricentis.com/Obstacles/41037')
+        cy.get('tbody > :nth-child(3) > :nth-child(5)').invoke('text').then((text) => {
```

Comment:
'tbody > :nth-child(3) > :nth-child(5)' is not an ideal solution. It will break if a new col/row added / removed.

Can you try a better way? Hint:

cy.get('#timeTable').within(
    // Get the first tr as header, compare each header name to find the col index.
    // Get the first col and find the index for the timespan.
)